internal/coremsgs/en_config_descriptions.go

```diff
@@ -92,6 +92,67 @@ var (
 
 	ConfigBlockchainFabricFabconnectProxyURL = ffc("config.blockchain.fabric.fabconnect.proxy.url", "Optional HTTP proxy server to use when connecting to Fabconnect", "URL "+i18n.StringType)
 
+	ConfigPluginDatabase     = ffc("config.plugins.database", "The list of configured Database Connectors.", i18n.StringType)
+	ConfigPluginDatabaseName = ffc("config.plugins.database[].name", "The name of the Database Connector.", i18n.StringType)
```

Comment:
I would say "the user-defined name for the database plugin instance".

I actually would not use "connector" at all in these contexts, as I think we use "connector" to mean the thing that is outside FireFly (not the code that is loaded inside FireFly).